### ISSUE TITLE
Update MongoDB Driver to support SCRAM-SHA-1 for authentication

### ISSUE THREAD

**Issue by dergraf** [erlio/vernemq#436]
The tag we're using is v0.7.9 and current tag is v3.0.2. They introduced several new features and most important support SCRAM-SHA-1 for authentication which will be the default for the new MongoDB version.

**Comment by PedroMD** [comment]
Any update on this? Having `SCRAM-SHA-1` auth mechanism only pretty much renders this integration useless as it is very very old, sadly...

**Comment by larshesel** [comment]
We'd gratefully accept this as a contribution - currently we have some other higher priority stuff which needs to be done before we get to this. I can't promise when we'll get around to this atm other than hopefully 'soon'.

**Comment by PedroMD** [comment]
Thank you @larshesel. Unfortunately, I can not help with this as I have 0 erlang knowledge. 

However, FYI, I did try to do it using redis, but it is also failing. I've created a new issue here: https://github.com/erlio/docker-vernemq/issues/19

**Comment by larshesel** [comment]
The mongodb driver was upgraded to a version supporting SCRAM on master. It will be part of the VerneMQ 1.2.1 release.

**Comment by anpjavatech** [comment]
Hi
I'm getting mongo connection issue while trying to do Auth using database.
Using Vernemq docker and Mongo 3.6.17 version.

![image](https://user-images.githubusercontent.com/6368801/80300315-2359ab00-87b9-11ea-9a7f-337ce83de201.png)

In vernemq Im getting the following log

![image](https://user-images.githubusercontent.com/6368801/80300327-45532d80-87b9-11ea-91b5-8ca960287931.png)

Any help is deeply appreciated.
